locations/spiders/lidl_de.py

```diff
@@ -40,43 +61,24 @@ def parse_hours(self, hours):
 
         return opening_hours.as_opening_hours()
 
-    def parse_details(self, response):
-
-        lidlShops = response.css(".ret-o-store-detail")
-
-        for shop in lidlShops:
-            shopAddress = shop.css(".ret-o-store-detail__address::text").extract()
-            street = shopAddress[0]
-            postalCode = shopAddress[1].split()[0]
-            city = shopAddress[1].split()[1]
-            openingHours = shop.css(".ret-o-store-detail__opening-hours::text").extract()
-            services = response.css(".ret-o-store-detail__store-icon-wrapper")[0]
-            link = services.css('a::attr("href")').get()
-            coordinates = link.split("pos.")[1].split("_L")[0]
-            latitude = coordinates.split("_")[0]
-            longitude = coordinates.split("_")[1]
-
-            properties = {
-                "ref": latitude + longitude,
-                "street_address": street,
-                "postcode": postalCode,
-                "city": city,
-                "lat": latitude,
-                "lon": longitude,
-            }
-
-            hours = self.parse_hours(openingHours)
-
-            if hours:
-                properties["opening_hours"] = hours
-
-            yield GeojsonPointItem(**properties)
-
     def parse(self, response):
+        data = response.body.decode("utf-8").split("Microsoft_Maps_Network_QueryAPI_3(")[1]
+        fixed_data = data.rstrip(data[-1])
+        jsonData = json.loads(fixed_data)
+        shops_of_the_area = jsonData["d"]["results"]
+
+        shop_property = {}
+
+        for shop in shops_of_the_area:
```

Comment:
You probably want to re-create this dict every time through the loop.

```suggestion

        for shop in shops_of_the_area:
            shop_property = {}
```